app/controllers/admin/volunteers.py

```diff
@@ -25,81 +23,71 @@ def getVolunteers(query):
 
     return json.dumps(searchUsers(query))
 
-@admin_bp.route('/event/<eventID>/manage_volunteers', methods=['POST'])
-def updateVolunteerTable(eventID):
-    try:
-        event = Event.get_by_id(eventID)
-    except DoesNotExist as e:
-        print(f"No event found for {eventID}")
-        abort(404)
+@admin_bp.route('/event/<eventID>/manage_volunteers', methods=['GET', 'POST'])
+def manageVolunteersPage(eventID):
+    """
+    Controller that handles POST and GET requests regarding the Manage Volunteers page.
 
-    volunteerUpdated = updateEventParticipants(request.form)
-    if volunteerUpdated:
-        flash("Volunteer table succesfully updated", "success")
-    else:
-        flash("Error adding volunteer", "danger")
-    return redirect(url_for("admin.manageVolunteersPage", eventID=eventID))
+    POST: updates the event participants for a particular event by calling 
+    updateEventParticipants on the form.
 
-@admin_bp.route('/event/<eventID>/manage_volunteers', methods=['GET'])
-def manageVolunteersPage(eventID):
+    GET: retrieves all necessary participant lists and dictionaries and categorizes
+    the participants/volunteers into their respective participation statuses. Then 
+    renders the manageVolunteers.html template.
+    """
     try:
         event = Event.get_by_id(eventID)
     except DoesNotExist as e:
         print(f"No event found for {eventID}", e)
```

Comment:
This print statement for debugging?

Reply:
I believe it was already in there I just moved it around. I think we do some other similar stuff other places in the code base to alert us when the parameters passed in on request fail.